applications/crowdloan_frontend_template.md

```diff
@@ -0,0 +1,189 @@
+# Crowdloan Front End Template
+
+- **Team Name:** 10Clouds Sp. z o.o.
+- **Payment Address:** 0xa79aa9572c7e1f8909fcc9674d676b73d5a71faa (aUSD)
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 2
+
+## Project Overview :page_facing_up:
+
+Responding to RFP: https://github.com/w3f/Grants-Program/blob/master/rfps/open/crowdloan_front_end_template.md
+
+### Overview
+
+A front-end white-label template for teams to use to easily build their Polkadot Crowdloan websites.
+
+### Project Details
+
+The overall objective of this project is to provide a white-label solution to teams to be able to plug into and offer all the information users need to contribute to their Crowdloan. In essence, it's a collection of:
+
+1. Project information
+2. Rewards Schema
+3. Current contributions
+4. Time left in Crowdloan and competition
+5. Contribute CTA
+6. After the Crowdloan
+
+All of the above should be done style agnostic so that the project can plug their own look and feel into the site.
+(as per RFP)
+
+### Ecosystem Fit
+
+Defined by RFP
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+- Product Delivery Manager - Jedrzej Wencka
+- Tech Lead - Aleksander Krasnicki
+- React - Maciej Topor
+- Design Lead - Krzysztof Juszczyk
+- UX & Product Designer - Paweł Poterała
+
+
+### Contact
+
+- **Contact Name:** Edgar Czop
+- **Contact Email:** edgar.czop@10clouds.com
+- **Website:** https://10clouds.com/
+
+### Legal Structure
+
+- **Registered Address:**  ul. Chmielna 73, 00-801 Warsaw, Poland
+- **Registered Legal Entity:** 10Clouds Sp. z o.o. (Ltd.)
+
+### Team's experience
+
+The team was involved in creating an internal NFT Marketplace white-label. As the name suggests it's a website that allows mint and trade NFTs between users. It is strongly inspired by OpenSea and follows the same lazy minting pattern. After login with Metamask user can customize the profile, can create collections, and start lazy-minting NFTs. The website contains extensive search/filters for collections and NFT. NFT can be created as Open for offers which means another user can offer a price for which they want to buy it and the owner can accept any of them. The other way of selling is the „Buy now" option which allows you to sell NFTs for a set, fixed price. In both scenarios, the NFT is minted during the sale process and transferred to the new owner.
```

Comment:
```suggestion
The team was involved in creating an internal NFT Marketplace white-label. As the name suggests it's a website that allows mint and trade NFTs between users. It is strongly inspired by OpenSea and follows the same lazy minting pattern. After login with Metamask the user can customize the profile, create collections, and start lazy-minting NFTs. The website contains extensive search/filters for collections and NFTs. An NFT can be created as _Open for offers_ which means another user can offer a price for which they want to buy it and the owner can accept any of them. The other way of selling is the „Buy now" option which allows you to sell NFTs for a set, fixed price. In both scenarios, the NFT is minted during the sale process and transferred to the new owner.
```

Reply:
I've added a link to the demo (https://marketplace.10clouds.io)